### DISCUSSION TITLE
Converting Record Chain of domain host to string results in .foo.com

### DISCUSSION THREAD

**Discussion by pusateri** [NLnetLabs/domain#252] [Q&A]
If I have a domain host entry from reading a zone file that looks like:
Chain { left: RelativeDname(), right: Dname(foo.com.) }, ZoneRecordData::A(A { addr: x.y.z.a })

and I convert the chain into a String with .to_string(), I get .foo.com instead of foo.com which is what I expected.

**Comment by partim** [comment]
Looks like that is broken – Chain is very naive about printing and simply outputs the left and the right part connected with a dot. I’ll provide a fix for it.

**Comment by partim** [comment]
The fix is in #253.
